js/main.js

```diff
@@ -1,23 +1,25 @@
 /* global data */
-
 const $imgSrcChange = document.querySelector('.changed');
 const $photoId = document.querySelector('.photoURL');
 const $form = document.querySelector('form');
+const $entriesList = document.querySelector('.unordered');
+const $anchorEntries = document.querySelector('.entriesBtn');
+const $entryForm = document.querySelector('.query_one');
+const $entriesBottom = document.querySelector('.query_two');
 
 $photoId.addEventListener('input', (e) => {
   $imgSrcChange.setAttribute('src', e.target.value);
 });
 // submit event
 
 $form.addEventListener('submit', (e) => {
-  e.preventDefault();
-
   const objStorageFormInputs = {
     entryId: data.nextEntryId,
     title: e.target.elements.title.value,
     photoID: e.target.elements.photoID.value,
     textarea: e.target.elements.textarea.value,
   };
+  e.preventDefault();
```

Comment:
This should be at the top of the function. We want this to be the first thing to run